### PR TITLE
[Merged by Bors] - feat(PNat/Basic): add three lemmas

### PR DESCRIPTION
**Pull request by rwst**
Three small lemmata in support of further PNat proofs

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/PNat/Basic.lean [hunk 1]
```diff
@@ -278,6 +278,21 @@ theorem pow_coe (m : ℕ+) (n : ℕ) : ↑(m ^ n) = (m : ℕ) ^ n :=
...
 #align pnat.pow_coe PNat.pow_coe
 
+/-- b is greater one if any a is less than b -/
+theorem one_lt_of_lt {a b: ℕ+} (hab: a < b): 1 < b := by
+  by_contra! hle
+  rw [le_one_iff] at hle
+  subst hle
+  have : ¬a < 1 := not_lt_one a
+  contradiction
+#align pnat.one_lt_of_lt PNat.one_lt_of_lt
```
Comment (grunweg) [inline Mathlib/Data/PNat/Basic.lean:288]: We only use #aligns for code that was in mathlib3. Please remove these again.

File: Mathlib/Data/PNat/Basic.lean [hunk 2]
```diff
@@ -278,6 +278,21 @@ theorem pow_coe (m : ℕ+) (n : ℕ) : ↑(m ^ n) = (m : ℕ) ^ n :=
   rfl
 #align pnat.pow_coe PNat.pow_coe
 
+/-- b is greater one if any a is less than b -/
+theorem one_lt_of_lt {a b: ℕ+} (hab: a < b): 1 < b := by
+  by_contra! hle
+  rw [le_one_iff] at hle
+  subst hle
+  have : ¬a < 1 := not_lt_one a
+  contradiction
```
Comment (YaelDillies) [inline Mathlib/Data/PNat/Basic.lean:287]:
```suggestion
theorem one_lt_of_lt {a b: ℕ+} (hab : a < b) : 1 < b := bot_le.trans_lt hab
```

File: Mathlib/Data/PNat/Basic.lean [hunk 3]
```diff
@@ -278,6 +278,21 @@ theorem pow_coe (m : ℕ+) (n : ℕ) : ↑(m ^ n) = (m : ℕ) ^ n :=
   rfl
 #align pnat.pow_coe PNat.pow_coe
 
+/-- b is greater one if any a is less than b -/
+theorem one_lt_of_lt {a b: ℕ+} (hab: a < b): 1 < b := by
```
Comment (grunweg) [inline Mathlib/Data/PNat/Basic.lean:282]: Please put spaces around the colon. (Did you see the [style guide](https://leanprover-community.github.io/contribute/style.html) already? It explains this and many other things - by reading and following it, you can make your code get accepted faster - as reviewers don't have to point this out :-))
```suggestion
theorem one_lt_of_lt {a b : ℕ+} (hab : a < b) : 1 < b := by

**Review by grunweg — COMMENTED** [review]
Thanks for your contribution to mathlib! I have a few small comments, mostly about style.
I did not look at the proof carefully.

**Comment by grunweg** [comment]
Please mark your code as `awaiting-review` when you addressed the comments.

File: Mathlib/Data/PNat/Basic.lean [hunk 4]
```diff
@@ -278,6 +278,13 @@ theorem pow_coe (m : ℕ+) (n : ℕ) : ↑(m ^ n) = (m : ℕ) ^ n :=
   rfl
 #align pnat.pow_coe PNat.pow_coe
 
+/-- b is greater one if any a is less than b -/
+theorem one_lt_of_lt {a b : ℕ+} (hab : a < b) : 1 < b := bot_le.trans_lt hab
+
+theorem add_one (a : ℕ+) : a + 1 = succPNat a := rfl
+
+theorem lt_succ_self (a : ℕ+) : a < succPNat a := lt.base a
```
Comment (urkud) [inline Mathlib/Data/PNat/Basic.lean:286]: While you're at it, could you please add lemmas for `a < a + b` (`lt_add_right`) and `b < a + b` (`lt_add_left`) as well? The proofs can reuse lemmas about `Nat`. Thank you!

bors d+

Comment (rwst) [reply Mathlib/Data/PNat/Basic.lean:286]: Will do later. I'm not done with `PNat`.

**Comment by rwst** [comment]
bors merge

File: Mathlib/Data/PNat/Basic.lean (hunk 4 above)
Comment (rwst) [reply Mathlib/Data/PNat/Basic.lean:286]: @urkud They are already in PNat:
https://github.com/leanprover-community/mathlib4/blob/b119a280b5e1ea54d4b78fff2cedbbf5d3a4d81b/Mathlib/Data/PNat/Basic.lean#L251

**Pull request closed without merging**